scipy/optimize/tests/test_quadratic_assignment.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+import numpy as np
+from numpy.testing import assert_array_equal
+from scipy.optimize import quadratic_assignment
+
+
+@pytest.mark.parametrize("n", [10, 100])
+@pytest.mark.parametrize("directed", [False, True])
+def test_weighted_graph_isomorphism(n, directed):
```

Comment:
I'll combine these tests with those from gh-11862, so no need to check that everything is covered.